### PR TITLE
Support more style range options

### PR DESCRIPTION
**Pull request by manthey**

### CODE REVIEW

File: large_image/tilesource/base.py [hunk 1]
```diff
@@ -1194,6 +1218,13 @@ def _applyStyle(self, image, style, x, y, z, frame=None):  # noqa
                 else:
                     output[:, :, channel] = numpy.maximum(
                         output[:, :, channel], numpy.where(keep, clrs, 0))
+        if dtype == 'uint16':
+            output = (output * 65535 / 255).astype(numpy.uint16)
+        elif dtype == 'float':
+            output /= 255
+        print(axis, output.shape)
```
Comment (banesullivan) [inline large_image/tilesource/base.py:1225]: remove print statement?

Comment (manthey) [reply large_image/tilesource/base.py:1225]: Thanks.

**Comment by banesullivan** [comment]
I'm curious when `full` is wanted/needed?

**Comment by banesullivan** [comment]
I suppose I've had a few `float` rasters but the data range was always much larger than 255 and thus the tiles would come back looking a single color. Setting `full` to 1 might still have the same issue?

**Comment by banesullivan** [comment]
All makes sense! Thanks for the explanation!

**Pull request merged**